integration/specs/Sidebar/sidebar-1.spec.js

```diff
@@ -0,0 +1,16 @@
+const PageSidebar = require('../../../src/components/Sidebar/pageObject');
+
+const SIDEBAR = '#sidebar-1';
+
+describe('Sidebar with multiple items', () => {
+    beforeEach(() => {
+        browser.url('/#!/Sidebar/1');
+        browser.refresh();
+    });
+    it('should make componet active on click event', () => {
+        const sidebar = new PageSidebar(SIDEBAR);
+        const sidebarItem = sidebar.getItem(1);
+        sidebarItem.click();
+        expect(sidebarItem.isActive()).toBe(true);
+    });
```

Comment:
here we can add another test:
'should make the second item active when first item is active and press TAB and ENTER key'
- click on the first item
- press TAB
- pres ENTER
- expect that second item is active